src/view_uniforms.rs

```diff
@@ -25,19 +26,23 @@ pub struct OutlineViewBindGroup {
 
 pub fn extract_outline_view_uniforms(
     mut commands: Commands,
-    query: Extract<Query<(Entity, &Camera), With<Camera3d>>>,
+    query: Extract<Query<(Entity, &Camera, Option<&RenderLayers>), With<Camera3d>>>,
 ) {
-    for (entity, camera) in query.iter() {
+    for (entity, camera, view_mask) in query.iter() {
         if !camera.is_active {
             continue;
         }
         if let Some(size) = camera.logical_viewport_size() {
-            commands
-                .get_or_spawn(entity)
+            let mut entity_commands = commands.get_or_spawn(entity);
+            entity_commands
                 .insert(OutlineViewUniform { scale: 2.0 / size })
                 .insert(RenderPhase::<StencilOutline>::default())
                 .insert(RenderPhase::<OpaqueOutline>::default())
                 .insert(RenderPhase::<TransparentOutline>::default());
+
+            if let Some(view_mask) = view_mask {
+                entity_commands.insert(*view_mask);
```

Comment:
`RenderLayers` is part of the public Bevy API, designed to let downstream developers customize the rendering behaviors, so if it's removed upstream it would be a major breaking change in Bevy. Also `OutlineRenderLayers` is just a newtype of `RenderLayers` so replacing this `RenderLayers` with `OutlineRenderLayers` wouldn't especially spare us any harm from the removal of `RenderLayers`.

I think the best justification to change this from `RenderLayers` to `OutlineRenderLayers` would be if we anticipate that users would want to completely isolate the meanings of their render layer values for outlines from the meanings that are used for meshes. I'm somewhat biased against this because I've already set up my application so that render layers values mean the same thing for both meshes and their outlines, and making this change would mean users need to remember to manage both `RenderLayers` and `OutlineRenderLayers` separately for each of their views, which strikes me as redundant.

I don't think it would be "wrong" to use `OutlineRenderLayers` for views so if this is something you feel strongly about then I don't mind making the change. I just personally find it a little bit unintuitive.

Reply:
Ah, thanks for elaborating. Now I understand your concern. I do think it's a stretch (especially the idea that their extracted `RenderLayers` would have a different meaning or value than the main world's `RenderLayers`) but I'll admit that we can't guarantee it won't happen :grimacing: 

I'm personally okay with the risk for the reasons that I mentioned earlier (I believe that leveraging the camera's ordinary `RenderLayers` is the least surprising behavior for users), but I won't object if you'd like me to change it.

Reply:
It was worth sounding out, but I think we're good.